interactive_cli.py

```
@@ -0,0 +1,66 @@
+"""Simple interactive menu for My Manga Recap."""
+
+import sys
+from typing import Optional
```

Flake8 (7.2.0)

[error] 'typing.Optional' imported but unused

(F401)